js/interface.js

```diff
@@ -44,13 +44,14 @@ function oembed(options) {
       var notSupported = ['video', 'link'].indexOf(response.type) === -1;
 
       if (response.thumbnail_url) {
-        if (!response.width) {
+        if (!response.width || response.thumbnail_width) {
```

Comment:
Why is the `!response.width` part of the logic necessary?

Reply:
without !response.width and !response.height logic I tested and it works fine. so we can also remove this condition and keep only the thumbnail width condition.

Reply:
@amolsuryawanshi47 Did you have a reason for keeping the `!response.width` part?

Reply:
we thought to keep this because we considered the response.width and response.height may not be present but after your comment on this we got a clear idea about it and we always get the thumbnail_width and  thumbnail_height in response. So we think to remove this condition.